src/containers/Redemptions.jsx

```diff
@@ -1,26 +1,119 @@
 import React from 'react';
+import { connect } from 'react-redux';
 
+// third party libraries
+import PropTypes from 'prop-types';
+
+// components
 import ActivityCard from '../components/redemptions/RedemptionCard';
 import Page from './Page';
 import PageHeader from '../components/header/PageHeader';
 import MasonryLayout from '../containers/MasonryLayout';
 import Stats from '../components/sidebar/Stats';
+
+// thunk
+import { fetchRedemption } from '../actions/redeemPointsAction';
+
+// helpers
 import dateFormatter from '../helpers/dateFormatter';
-import activities from '../fixtures/activities';
-import stats from '../fixtures/stats';
 import filterActivities from '../helpers/filterActivities';
 
+// fixtures
+import stats from '../fixtures/stats';
+
 class Redemptions extends React.Component {
+  /**
+   * @name defaultProps
+   */
+  static defaultProps = {
+    societyId: '',
```

Comment:
The society Id will often be a string. I personally like to assume that it is a `-1` since it pretty much matches a Number schema. Open to discuss this.